Tests/MatterControl.Tests/MatterControl/MatterControlUtilities.cs

```diff
@@ -824,7 +824,7 @@ public static void WaitForCommunicationStateDisconnected(this AutomationRunner t
 				testMethod,
 				maxTimeToRun,
 				defaultTestImages,
-				closeWindow: () =>
+				closeWindow: (Action)(() =>
```

Comment:
Unless this form is required, we should revert this back to the original format

@jlewin  - ping